libdevice/fallback-cassert.cpp

```diff
@@ -9,10 +9,41 @@
 #include "wrapper.h"
 
 #ifdef __SPIR__
+
+struct AssertHappened {
+  int Flag = 0;
+};
+
+#ifndef __SYCL_GLOBAL_VAR__
+#define __SYCL_GLOBAL_VAR__ /*__attribute__((sycl_global_var))*/
+#endif
+
+namespace cl {
```

Comment:
`libdevice` is a SPIR-V device library, so it seems inconvenient that we have to use the SYCL namespaces here.  Can you please explain why you need to declare `AssertHappenedMem` in `cl::sycl::detail`?

Reply:
The variable was put into namespace to show that it's for SYCL.
If namespace use is inconvenient here, I'll change it's naming to `__SYCL_AssertHappenedMem` w/o any namespace.